src/VectorAffineFunctionAsMatrix.jl

```diff
@@ -21,6 +21,8 @@ function MOI.output_dimension(v::VectorAffineFunctionAsMatrix)
     return size(v.matrix, 1)
 end
 
+to_vaf(tape::SPARSE_VECTOR) = tape
```

Comment:
I've wanted this also, but I skipped it since `vaf` was supposed to mean VectorAffineFunction. Maybe we can do this + rename it?

Reply:
I'm going to leave this hack for now, and rename it in a separate PR. `to_vaf` is really `_to_vector_moi`